app/controllers/planning_application/review_permitted_development_rights_controller.rb

```diff
@@ -50,5 +52,23 @@ def permitted_development_right_params
     def status
       save_progress? ? "review_in_progress" : "review_complete"
     end
+
+    def set_page_title
+      @page_title =
+        if @planning_application.possibly_immune?
+          "Immunity/permitted development rights"
+        else
+          "Permitted development rights"
+        end
+    end
+
+    def set_page_heading
+      @page_heading =
+        if @planning_application.possibly_immune?
+          "Review immunity/permitted development rights"
+        else
+          "Check permitted development rights"
+        end
+    end
```

Comment:
Could we not just put this in a helper instead, why instantiate variables ?

Reply:
I guess the only thing about using a helper instead is it will have to calculate it three times on the page rather than once in the controller. But I can see the benefit of not having loads of logic in hte controller